tasks/download.yml

```diff
@@ -16,24 +26,4 @@
     owner: "{{ odoo_role_odoo_user }}"
     group: "{{ odoo_role_odoo_group }}"
     extra_opts: [--strip-components=1]
-    creates: "{{ odoo_role_odoo_path }}/setup.py"
-  when: odoo_role_odoo_edition  == "odoo"
-
-# Odoo from the OCA/OCB repository: https://github.com/OCA/OCB
-- name: "Git clone Odoo branch {{ odoo_role_odoo_version }}"
-  become_user: "{{ odoo_role_odoo_user }}"
-  git:
-    repo: "{{ odoo_role_odoo_git_url }}"
-    dest: "{{ odoo_role_odoo_path }}"
-    version: "{{ odoo_role_odoo_version }}"
-    depth: 1
-  when: odoo_role_odoo_edition == "oca"
-
-- name: "Git checkout to the head {{ odoo_role_odoo_head }}"
-  become_user: "{{ odoo_role_odoo_user }}"
-  git:
-    clone: no
-    repo: "{{ odoo_role_odoo_git_url }}"
-    dest: "{{ odoo_role_odoo_path }}"
-    version: "{{ odoo_role_odoo_head }}"
-  when: odoo_role_odoo_edition == "oca"
+    #creates: "{{ odoo_role_odoo_path }}/setup.py"
```

Comment:
I'm still not sure if using `creates` is a good idea, we added it to avoid `unarchive` to run if the archive has been uncompressed already... but it doesn't cover updates...